### DISCUSSION TITLE
List of supported roms?

### DISCUSSION THREAD

**Discussion by ScottN13** [Mahmud0808/Iconify#125] [Q&A]
Is there a list of supported roms? If so, is OneUi 4.1 supported?

**Comment by Mahmud0808** [comment]
OneUI, OOS, ColorOS, MIUI is not supported.

Currently supported roms:
- Pixel Stock ROM
- AOSP Based Custom ROMs

ROMs that work but no support from me:
- Moto Stock ROM
- XPeria Stock ROM
- Nothing OS (Mostly works)

**Reply by ScottN13** [reply]
So Pixel Experience is supported since its AOSP?

**Reply by Mahmud0808** [reply]
Yes it's supported.

**Reply by ScottN13** [reply]
Alright, thanks!

**Reply by huixinzhilian** [reply]
Battery icon does not work in nothing os

**Reply by ScottN13** [reply]
> Battery icon does not work in nothing os

They have few support from Iconify dev. Sorry.
